src/features/SwapAssets/stateFormFactory.tsx

```diff
@@ -34,51 +34,26 @@ const swapFormInitialState = {
   lastChangedAmount: LAST_CHANGED_AMOUNT.FROM
 };
 
-const BASE_ASSET_ADDRESS = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee';
-
 const SwapFormFactory: TUseStateReducerFactory<SwapFormState> = ({ state, setState }) => {
   const network = useSelector(selectNetwork(state.selectedNetwork)) as Network;
+  const baseAsset = useSelector(getBaseAssetByNetwork(network));
+  const assets = useSelector(getSwapAssetsByNetwork(state.selectedNetwork));
+  const sortedAssets = assets.sort((asset1, asset2) =>
+    (asset1.ticker as string).localeCompare(asset2.ticker)
+  );
```

Comment:
Why is `asset1.ticker` not properly inferred here? Is this an issue with the selector again?

Reply:
This was copied from elsewhere, the types can now be inferred. Will fix